src/main/java/org/quiltmc/syntaxpain/QuickFindDialog.java

```diff
@@ -95,7 +97,7 @@ public void windowLostFocus(WindowEvent e) {
 				target.requestFocus();
 			}
 		};
-		this.addWindowFocusListener(focusListener);
+		// this.addWindowFocusListener(focusListener);
```

Comment:
I would rather this just be removed instead of commented

Reply:
Oh whoops, I commented it out to test without the dialog vanishing

Reply:
Should I remove it tho?

Reply:
I think it should stay

Reply:
fixed it